### ISSUE TITLE
Transition Option is ignored

### ISSUE THREAD

**Issue by kdlslyv** [Maronato/vue-toastification#54]
**Versions**
List of versions known to be affected.
- 1.2.2

0.8 worked fine

**Describe the bug**
you can put any of the other transitions in the options e.g:
```js
const options = {
  draggable: false,
  transition: "Vue-Toastification__fade"
}
```
it gets ignored

**Expected behavior**
less intrusive fade transition is applied

**Demo**
- git cloned this repo
- used vue example
- modify options like above; it won't work.

However:
If you manually update the settings as documented like this:
```js
      const update = {
      transition: "Vue-Toastification__fade"
           };
      this.$toast.updateDefaults(update);
```
it works.

**Additional context** 
This plugin is the best documented, feature-rich and most comfortable for showing toasts, great work!

**Comment by Maronato** [comment]
Thanks for the bug report! I've found the issue and the fix will be released with 1.2.3 in a couple of minutes

**Comment by Maronato** [comment]
I'm also glad you enjoy the plugin :)
If you find any other bugs or feel like there is a feature missing, don't hesitate to create an issue.
